app/services/search_query_sanitization_service.rb

```diff
@@ -4,7 +4,7 @@ class SearchQuerySanitizationService
   ANYTHING_EXCEPT_NUMBERS_AND_SPACE_REGEX = /[^[[:digit:]] ]+/.freeze
   ANYTHING_EXCEPT_NUMBERS_SPACE_PERIOD_REGEX = /[^[[:digit:]]. ]+/.freeze
   ANYTHING_EXCEPT_NUMBERS_REGEX = /[^[[:digit:]]]+/.freeze
-  ANYTHING_EXCEPT_LETTERS_NUMBERS_HYPHEN_SPACE_REGEX = /[^[[:alnum:]] -]+/.freeze
+  ANYTHING_EXCEPT_LETTERS_NUMBERS_COMMON_PUNCTUATION_REGEX = /[^[[:alnum:]] ā ē ī ō ū Ā Ē Ī Ō Ū \- () ']+/.freeze
```

Comment:
I think you probably only need one space in this character group?

Reply:
oh true why did I do it like that for?

Reply:
```suggestion
  ANYTHING_EXCEPT_LETTERS_NUMBERS_COMMON_PUNCTUATION_REGEX = /[^[[:alnum:]]āēīōūĀĒĪŌŪ\-() ']+/.freeze
```